packages/ketcher-react/src/script/ui/views/components/ContextMenu/hooks/useAddAttachmentPoint.ts

```diff
@@ -80,6 +80,7 @@ const useAddAttachmentPoint = () => {
 
       editor.update(action);
       editor.selection(null);
+      editor.focusCliparea();
```

Comment:
Don't we need to call this function in packages/ketcher-react/src/script/ui/views/components/ContextMenu/hooks/useRemoveAttachmentPoint.ts also?